tools/manage.ps1

```diff
@@ -0,0 +1,279 @@
+<#
```

Comment:
Few notes:
- This does not belong to this PR.
- I don't think it belongs to `tools` subfolder, but should be in root, if we add it.
- The template is not used only by Ynput team, it is for community addons too.